intuita-webview/src/CreateIssue/index.tsx

```diff
@@ -7,75 +7,73 @@ import {
 import { useEffect, useState } from 'react';
 import { vscode } from '../shared/utilities/vscode';
 import styles from './style.module.css';
-import { IssueFormData } from '../../../src/components/webview/webviewEvents';
 import './tiptap.css';
 
 type Props = Readonly<{
 	title: string;
 	body: string;
 }>;
 
-const CreateIssue = ({ title, body }: Props) => {
+const CreateIssue = (props: Props) => {
 	// TODO: handle loading for creating issue
 	const [loading] = useState(false);
-	const [formData, setFormData] = useState<IssueFormData>({
-		title,
-		body,
-	});
 
-	useEffect(() => {
-		setFormData((prevData) => ({
-			...prevData,
-			title,
-			body,
-		}));
-	}, [title, body]);
+	const [title, setTitle] = useState('');
 
-	const onChangeFormField =
-		(fieldName: keyof IssueFormData) =>
-		(e: Event | React.FormEvent<HTMLElement>) => {
-			const value = (e as React.ChangeEvent<HTMLInputElement>).target
-				.value;
-			setFormData((prevData) => ({
-				...prevData,
-				[fieldName]: value,
-			}));
-		};
+	const onChangeTitle = (e: Event | React.FormEvent<HTMLElement>) => {
+		const value =
+			'target' in e && e.target !== null
+				? (e.target as HTMLTextAreaElement).value
+				: null;
+		if (value === null) {
+			return;
+		}
+		setTitle(value);
+	};
 
 	const handleSubmit = (e: React.FormEvent) => {
 		e.preventDefault();
 
+		if (editor === null) {
+			return;
+		}
+
 		vscode.postMessage({
 			kind: 'webview.sourceControl.createIssue',
-			data: formData,
+			data: {
+				title,
+				body: editor.getHTML(),
+			},
 		});
 	};
 
 	const extensions = [StarterKit];
 
-	const content = formData.body;
-
 	const editor = useEditor({
 		extensions,
-		content,
+		content: props.body,
 		editable: true,
-		onUpdate: ({ editor }) => {
-			setFormData((prevData) => ({
-				...prevData,
-				body: editor.getHTML(),
-			}));
-		},
-		autofocus: 'end',
 	});
 
+	useEffect(() => {
+		setTitle(props.title);
+	}, [props.title]);
+
+	useEffect(() => {
+		if (editor === null) {
+			return;
+		}
+		editor.commands.setContent(props.body, false);
```

Comment:
`editor?.commands.setContent(props.body, false);`

Reply:
```
if (editor === null) {
			return;
		}
```
isn't this enough?

Reply:
you don't need to use that `if` statement with `?.` 3 lines saved.

Reply:
addressed